x-pack/plugins/apm/public/components/app/settings/general_settings/index.tsx

```diff
@@ -118,7 +115,8 @@ export function GeneralSettings() {
               links: docLinks.links.management,
               showDanger: (message: string) =>
                 notifications.toasts.addDanger(message),
-              validateChange: async () => settingsValidationResponse,
+              validateChange: async (key: string, value: any) =>
```

Comment:
I think you don't need to add `async` here, `settings.client.validateValue` returns a promise already.

```suggestion
              validateChange: (key: string, value: any) =>
```

Reply:
Nice catch! Fixed in [bd9b76c](https://github.com/elastic/kibana/pull/175874/commits/bd9b76c2a1fbd3f0c0c4b1baa1803f82b2406c9b).